vivisect/impemu/emulator.py

```diff
@@ -264,7 +264,7 @@ def stepi(self):
         if not self.checkCall(starteip, endeip, op):
             self.checkBranches(starteip, endeip, op)
 
-    def runFunction(self, funcva, stopva=None, maxhit=None, maxloop=None):
+    def runFunction(self, funcva, stopva=None, maxhit=None, maxloop=None, ignoreunknown=False):
```

Comment:
This method's args are starting to get a bit crazy...  how about the emu adds self.strictops = True ( or something ) and we make the behavior emulator instance wide?